app/controllers/codelijsten/index.js

```diff
@@ -1,10 +1,21 @@
 import Controller from '@ember/controller';
 import { service } from '@ember/service';
+import { action } from '@ember/object';
 
 export default class CodelijstenController extends Controller {
   @service features;
+  @service router;
 
   sort = '-preflabel';
   page = 0;
   size = 20;
+
+  @action
+  returnToFormBuilder() {
```

Comment:
Would name it a bit more specific. "returntoFormBuilder()" and the next line will add an id in the route. Maybe just go with "goToFormWithId" or "routeToOpenForm" or "routeToLastOpenedForm"